### ISSUE TITLE
Plugin overrides statusBaroverlaysWebView() setting in iOS

### ISSUE THREAD

**Issue by schlackl** [admob-plus/admob-plus#321]
Steps to reproduce:
1. Install the admob-plus-cordova

2. Include the following in app initialize for deviceready
admob
   .start()
   .then(() =>
         admob.BannerAd.config({
             marginTop: 10,
             marginBottom: 10,
         }),
   )
   .catch(alert);

if (StatusBar != null) {
	StatusBar.overlaysWebView(false);
	StatusBar.styleDefault();
}

3. Add the following to document.ready()
var objBannerAd = new admob.BannerAd({
   adUnitId: admobid.banner,
});
objBannerAd.show();

4. When the app loads, the statusBar is overlayed as it should be but once AdMob has initialized and calls show() the statusBar style is reverted/changed to be overlaysWebView(true) and there is no way to change it back.

Honestly, this bug has existed for a while and I had moved to another admob plugin that resolves the issue however that plugin no longer builds on Android due to some Java errors so I've come back to this one which works great on iOS and Android but this particular issue is a big one and I have no idea how to resolve or work-around it.

I am building using Ionic Appflow with the following build stack:
Cordova CLI | 10.0.0
Ionic CLI | 6.13.1
Node.js | 14.15.0
npm | 6.14.8
Yarn | 1.22.10
macOS | 10.15.7
Carthage | 0.36.0
CocoaPods | 1.10.0
Xcode | 12.1

I'm hopeful you are able to see the issue and resolve it - this plugin is great otherwise!

**Comment by ratson** [comment]
@schlackl Did you try without `admob.BannerAd.config({
marginTop: 10,
marginBottom: 10,
}),`?

**Comment by schlackl** [comment]
> @schlackl Did you try without `admob.BannerAd.config({ marginTop: 10, marginBottom: 10, }),`?

Yes I tried that and while it seems to work at first, when I return from the camera to my app after taking a photo for use in my app, the statusbar reverts to overlay my app and again. However, for some reason if I "pause" the app into the background and then resume the app, the statusbar fixes itself.

That said, it is definitely the admob plugin that changes the statusbar style after returning from the camera because if I remove that from the app and test I do not have that issue with the statusbar style reverting to overlay my app.

So it seems it's an issue when coming back from the camera to the app. Any other thoughts on what is happening here?

**Comment by ratson** [comment]
@schlackl Is the original issue solved in `1.8.3`?
For the camera issue, would you create a test repo for me to check.

**Comment by schlackl** [comment]
> @schlackl Is the original issue solved in `1.8.3`?
> For the camera issue, would you create a test repo for me to check.

It's difficult to say if the original issue is fixed - it might be the same issue but slight disguised - I'm working on a couple other tests to be sure....I'll post here again when I have more - thanks for the quick replies!

**Comment by schlackl** [comment]
Hello - this seems to be resolved and is working. There was a piece of code left over from the previous plugin that worked around a status bar issue but after removing that things seem to be working correctly now. If other issues arise I will open a new case but this is closed now :)
